errorMSG

```diff
@@ -0,0 +1,8 @@
+Hey, sorry for the awkward way of letting you know, but I found an error on minimalist.
```

Comment:
No worries! :) 

This isn't the right repo (it isn't public) but I'll improve the error messages so they are easier for people to understand.